src/main/webapp/app/code-editor/code-editor.component.ts

```diff
@@ -151,18 +155,27 @@ export class CodeEditorComponent implements OnInit, OnChanges, OnDestroy {
         this.latestResult = $event.newResult;
     }
 
+    /**
+     * Check if the received build logs are recent and format them for use in the ace-editor
+     * @param buildLogs
+     */
     updateLatestBuildLogs(buildLogs: BuildLogEntry[]) {
-        this.buildLogErrors = buildLogs
-            .map(({ log }) => log && log.match(this.errorLogRegex))
-            .filter(log => !!log && log.length === 6 && log[1] === 'ERROR')
-            .map(([, , fileName, row, column, text]) => ({
-                type: 'error',
-                fileName,
-                row: Math.max(parseInt(row, 10) - 1, 0),
-                column: Math.max(parseInt(column, 10) - 1, 0),
-                text: safeUnescape(text)
-            }))
-            .reduce((acc, { fileName, ...rest }) => ({ ...acc, [fileName]: [...(acc[fileName] || []), rest] }), {});
+        const ts = buildLogs.length ? Date.parse(buildLogs[0].time) : 0;
```

Comment:
I guess ts stands for timestamp, right? I would prefer if you could write timestamp instead to make it easier to read it

Reply:
True - I'll refactor this.

Reply:
Refactored.